foundations/powerPlatform/readme.md

```diff
@@ -203,6 +203,9 @@ The following section describes the design considerations and the design recomme
 - Environments can be created with - or without Dynamics 365 application templates available in the tenant.
 - Dynamics 365 applications takes a dependency on environments with Dataverse provisioned.
 - Dataverse for Microsoft Teams can be created directly from the Teams client, which grants the creator Owner permission on the Environment.
+- Managed Environments requires that the users are licensed, as all apps and flows in a managed environment are premium.
+- To enable Managed Environments, you must be a global admin, Power Platform admin, or Dynamics 365 admin in Azure Active Directory.
+- In the Power Platform admin center, Environment Admins are not allowed to activate managed environments. Environment Admins are able to see the 'Enable Managed Environment' action in the environment details page.
```

Comment:
There is a backlog item to address the issue of Environment Admins are able to see the 'Enable Managed Environment' action in the environment details page. Once Completed the "view" should be trimmed to only Role Admins.